src/runtime/hexagon_host.cpp

```diff
@@ -208,9 +245,8 @@ WEAK int halide_hexagon_initialize_kernels(void *user_context, void **state_ptr,
 
     // Create the module itself if necessary.
     if (!(*state)->module) {
-        debug(user_context) << "    halide_remote_initialize_kernels -> ";
         halide_hexagon_handle_t module = 0;
-        result = remote_initialize_kernels(code, code_size, &module);
+        result = remote_initialize_kernels(code, code_size, use_shared_object, &module);
```

Comment:
To clarify, "remote_initialize_kernels_v1" is a function pointer that points to the old `halide_hexagon_remote_initialize_kernels` function. For sanity, maybe it would be better to name the two function pointers "remote_initialize_kernels_v2" and "remote_initialize_kernels", for the current version.